subsys/bluetooth/controller/Kconfig

```diff
@@ -1197,6 +1209,17 @@ config BT_CTLR_ASSERT_HANDLER
 	  and will be invoked whenever the controller code encounters
 	  an unrecoverable error.
 
+config BT_CTLR_ASSERT_OPTIMIZE_FOR_SIZE
+	bool "Assertions optimized for code size"
+	default y if CPU_CORTEX_M
```

Comment:
Is this supported by all architectures, or should there be one or more `depends on` here?